src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,10 +65,21 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
-		$this->params = shortcode_atts( array(
-			'limit' => -1,
+		/**
+		 * Allow filtering of the default limit for the [tribe-user-event-confirmations] shortcode.
+		 * @since TBD
+		 *
+		 * @param int $default_limit The default limit to use.
+		 */
+		$default_limit = apply_filters( 'tribe_tickets_shortcodes_attendee_list_limit', 100 );
+
+		$this->params = shortcode_atts( [
+			'limit' => $default_limit,
 			'user'  => get_current_user_id()
-		), $params, $this->shortcode_name );
+		], $params, $this->shortcode_name );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: e1894c19c8057fb399f7294aabf38c3f</sub>